app/models/cloud_volume.rb

```diff
@@ -37,6 +37,15 @@ def self.class_by_ems(ext_management_system)
     ext_management_system && ext_management_system.class::CloudVolume
   end
 
+  def self.my_zone(ems)
+    # TODO(pblaho): find unified way how to do that
+    ems ? ems.my_zone : MiqServer.my_zone
+  end
+
+  def my_zone
+    self.class.my_zone(ext_management_system)
+  end
+
```

Comment:
Pulled this from CloudVolumeSnapshot, I figured they would want the same semantics here